src/Core/Misc/ExtensionMethods.cs

```diff
@@ -599,6 +599,17 @@ public static T[] ToArray<T>(this Newtonsoft.Json.Linq.JArray j)
             return retVal;
         }
 
+        /// <summary>
+        /// Convert Element Property value to Size
+        /// </summary>
+        /// <param name="property"></param>
+        /// <returns>A Size object</returns>
+        public static Size ToSize(this A11yProperty property)
+        {
+            if (property.Value is null) return Size.Empty;
```

Comment:
This throws a `NullReferenceException` if `property` is null:
```suggestion
            if (property?.Value is null) return Size.Empty;
```